lib/audit_test.pm

```diff
@@ -46,79 +46,81 @@ our $baseline_file = 'baseline_run.log';
 sub run_testcase {
     my ($testcase, %args) = @_;
 
+    if ($args{set_limit_interval}) {
+        assert_script_run('sed -i \'/\[Unit\]/a\StartLimitIntervalSec=0\' /usr/lib/systemd/system/auditd.service');
+        assert_script_run('systemctl daemon-reload');
+    }
+    if ($args{set_audit_rule}) {
+        assert_script_run('sed -i \'s/-a task,never/#-a task,never/\' /etc/audit/rules.d/audit.rules');
+        assert_script_run('systemctl restart auditd.service');
+    }
+
```

Comment:
It is not proper to put these code in this sub function as they are common set up steps for auditd, could you please put them in a new sub function?

Reply:
> I'd suggest keep the 2nd one here, since the rules is for syscall only (IMO)

Sure, if they are syscall only  then I would suggest to move the code to "syscalls.pm" as `run*` is designed for being invoked by all test cases, etc, crypto/netfileter/..., .